src/extension/messages/platformRequest/saveCellExecution.ts

```diff
@@ -187,7 +187,7 @@ export default async function saveCellExecution(
           },
         },
       }
-      const result = await graphClient.mutate(mutation)
+      const result = await graphClient.mutate(mutation as MutationOptions)
```

Comment:
I am concerned about this because I have no idea why this is needed.  The typescript error on this was very hairy.  If someone smarter than me knows what is going on here please let me know.

Reply:
Why would this code need to change at all if its running on head? What was the error you got?

Reply:
Argument of type '{ mutation: TypedDocumentNode<CreateExtensionCellOutputMutation, Exact<{ input: ReporterInput; }>>; variables: { ...; }; }' is not assignable to parameter of type 'MutationOptions<CreateExtensionCellOutputMutation, Exact<{ input: ReporterInput; }>, DefaultContext, ApolloCache<any>>'.
  The types of 'variables.input.notebook.cells' are incompatible between these types.
    Type '{ outputs: { items: CellOutputItem[]; metadata: { [key: string]: string; }; processInfo?: CellOutputProcessInfo | undefined; }[]; kind: CellKind; ... 4 more ...; executionSummary?: CellExecutionSummary | undefined; }[]' is not assignable to type 'ReporterCellInput[]'.
      Type '{ outputs: { items: CellOutputItem[]; metadata: { [key: string]: string; }; processInfo?: CellOutputProcessInfo; }[]; kind: CellKind; ... 4 more ...; executionSummary?: CellExecutionSummary; }' is not assignable to type 'ReporterCellInput'.
        Types of property 'executionSummary' are incompatible.
          Type 'CellExecutionSummary | undefined' is not assignable to type 'InputMaybe<ReporterCellExecutionSummaryInput> | undefined'.
            Type 'CellExecutionSummary' is not assignable to type 'ReporterCellExecutionSummaryInput'.
              Types of property 'executionOrder' are incompatible.
                Type 'number | undefined' is not assignable to type 'InputMaybe<UInt32> | undefined'.
                  Type 'number' has no properties in common with type 'UInt32'.ts(2345)

Reply:
Ok, after testing the wall thoroughly with my forehead I have found a loose brick. It seems this graphql stuff was generated and it seems from the timostam-ts proto type. On line 2014 in graphql.ts there exists a line `executionOrder?: InputMaybe<UInt32>;` which conforms to the timostam-ts type Notebook.Cell.CellExecutionSummary.executionOrder which is `executionOrder?: UInt32Value;` which is in contrast to the es type, which is, as you'd guess, a `number` ... so my guess is this graphql.ts needs to be regenerated.

Reply:
Yes I mean making sure your branch is up to date with upstream changes

Reply:
Can this graphql code continue to use the timeostamp protos in this PR?

Reply:
> Can this graphql code continue to use the timeostamp protos in this PR?

This brings us back to the "can we make these two coexist" question as I see it, which was attempted for quite a long time until it was determined they could not coexist, and the original GrpcSerializer was modified so that there would not be two implementations. From what I can tell this will need to be regenerated, or we can force cast (as I have done in this branch) and pray it doesn't cause any runtime errors.  I ran the tests for this and didn't get any failures, but I'm not sure this specific case is being tested. 

I don't see where graphql.ts gets generated.  Should I post a question on discord?

Reply:
Oh sorry I didn't realize from the diff this was using grpcSerializer. Yeah if its using grpcSerializer it might need to change. Sure ask in discord.